src/applications/mhv/secure-messaging/containers/App.jsx

```diff
@@ -81,7 +81,16 @@ const App = () => {
     trackLongTasks: true,
     defaultPrivacyLevel: 'mask-user-input',
   };
-  useDatadogRum(datadogRumConfig);
+  const userDetails = useMemo(
+    () => {
+      return {
+        loggedIn: user.login.currentlyLoggedIn,
+        accountUuid: user.profile.accountUUid,
+      };
+    },
+    [user.login.currentlyLoggedIn, user.profile.accountUUid],
```

Comment:
Suggestion: safely navigate these object properties with the `?.` operator.

Reply:
Thats a good call.